com.diy_fever.DIYLayoutCreator.json

```diff
@@ -5,7 +5,7 @@
     "command": "diylc",
     "sdk": "org.freedesktop.Sdk",
     "sdk-extensions": [
-        "org.freedesktop.Sdk.Extension.openjdk11"
+        "org.freedesktop.Sdk.Extension.openjdk17"
```

Comment:
You've switched to openjdk17 but forgot to also update the install script on line 98:

https://github.com/flathub/com.diy_fever.DIYLayoutCreator/blob/c9795a326e99c075aa10923fbfa24d816e793e56/com.diy_fever.DIYLayoutCreator.json#L98